openedx/core/djangoapps/content/learning_sequences/api/outlines.py

```diff
@@ -46,11 +48,11 @@ def get_course_outline(course_key: CourseKey) -> CourseOutlineData:
 
     See the definition of CourseOutlineData for details about the data returned.
     """
-    learning_context = _get_learning_context_for_outline(course_key)
```

Comment:
As long as the `LearningContext` model is preserved, I think it makes sense to have a separate `CourseContext` model to house the course-specific stuff. And since this call is specifically asking for a course outline, I think the shift here is appropriate as well.